src/misc/KeyManager.js

```diff
@@ -148,7 +148,7 @@ class KeyManager {
 				       return
 			       }
 			       let shortcuts = ((this._modalShortcuts.length
-				       > 1) ? this._modalShortcuts : this._shortcuts).concat(this._desktopShortcuts) // we do not want to show a dialog with the shortcuts of the help dialog
+				       > 1) ? this._modalShortcuts : this._shortcuts.map(s => s.shortcut)).concat(this._desktopShortcuts) // we do not want to show a dialog with the shortcuts of the help dialog
```

Comment:
Could we un-bork the formatting here? you didnt do it i know but since it's being touched anyway...